prow/knative/config.yaml

```diff
@@ -137,6 +137,7 @@ tide:
   queries:
   - orgs:
     - "knative"
+    - "knative-sandbox"
     excludedRepos:
```

Comment:
Shall we also remove the excludedRepos?

Reply:
No, the `excludedRepos` are archived repos that won't accept any more PRs.

Reply:
I verified we don't have any archived repos in `knative-sandbox`